
module.exports.throwErr = (err_message, log) => {
  throw new Error(err_message)
}